<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-78D4HDH" name="basicsTestNG.TestNGInvocation" tests="11" failures="0" timestamp="05 Oct 2018 01:39:33 GMT" time="3.061" errors="1">
  <testcase name="testA1" time="0.001" classname="basicsTestNG.TestNGInvocation"/>
  <testcase name="testA2" time="3.024" classname="basicsTestNG.TestNGInvocation">
    <error type="org.testng.internal.thread.ThreadTimeoutException" message="Method basicsTestNG.TestNGInvocation.testA2() didn&#039;t finish within the time-out 3000">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method basicsTestNG.TestNGInvocation.testA2() didn't finish within the time-out 3000
]]>
    </error>
  </testcase> <!-- testA2 -->
  <testcase name="testA1" time="0.002" classname="basicsTestNG.TestNGInvocation"/>
  <testcase name="testA1" time="0.019" classname="basicsTestNG.TestNGInvocation"/>
  <testcase name="testA1" time="0.001" classname="basicsTestNG.TestNGInvocation"/>
  <testcase name="testA1" time="0.002" classname="basicsTestNG.TestNGInvocation"/>
  <testcase name="testA1" time="0.002" classname="basicsTestNG.TestNGInvocation"/>
  <testcase name="testA1" time="0.002" classname="basicsTestNG.TestNGInvocation"/>
  <testcase name="testA1" time="0.004" classname="basicsTestNG.TestNGInvocation"/>
  <testcase name="testA1" time="0.002" classname="basicsTestNG.TestNGInvocation"/>
  <testcase name="testA1" time="0.002" classname="basicsTestNG.TestNGInvocation"/>
</testsuite> <!-- basicsTestNG.TestNGInvocation -->
